src/scroll-timeline-base.js

```diff
@@ -160,15 +154,23 @@ export function removeAnimation(scrollTimeline, animation) {
   animations.splice(index, 1);
 }
 
+function find(scrollTimeline, animation) {
+  const animations = scrollTimelineOptions.get(scrollTimeline).animations;
+  let index = animations.indexOf(animation);
+  return (index === -1) ? null : animations[index];
```

Comment:
The return value doesn't seem to be used - just the truthiness - should this be a contains method?